src/extensions/php-fpm-metrics/pkg/phpfpm/status.go

```diff
@@ -0,0 +1,69 @@
+package phpfpm
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/nginx/agent/v2/src/core"
+	log "github.com/sirupsen/logrus"
+)
+
+var shell core.Shell = core.ExecShellCommand{}
+
+// Status is an Enum that represents the status of PhpFpm.
+type Status int
+
+const (
+	UNKNOWN Status = iota
+	INSTALLED
+	RUNNING
+)
+
+// String get the string representation of the enum
+func (s Status) String() string {
+	switch s {
+	case INSTALLED:
+		return "INSTALLED"
+	case RUNNING:
+		return "RUNNING"
+	}
+	return "UNKNOWN"
+}
+
+// GetStatus returns phpfpm process status
+func GetStatus(pid, version string) Status {
+	output, err := shell.Exec("ps xao pid,ppid,command | grep 'php-fpm[:]'")
```

Comment:
this repo has a few features that depend on reading processes, could we re-use something like https://github.com/nginx/agent/blob/181b995a87a7c2905915f20acd0d2053c553ce64/src/core/process.go#L20C12-L20C12, or something else from <https://pkg.go.dev/github.com/shirou/gopsutil/v3/process>?

It's nice to avoid a shell execs if we can; they tend to be slower and can run into portability problems. A lot of times we can use a data structure directly instead of parsing stdout.

Reply:
In initial few MRs, I am putting forth on parity & functionality aspect by atomically adding components. These components at the moment are unit tested.
But I do plan to replace shell commands with `go-psutil` in future MRs where I would be integrating above atomic components, and in addition be testing on sandbox env.

Reply:
> I am putting forth on parity & functionality aspect

ok, so something like porting the python to go, then as another pass making it more efficient/idiomatic? That kind of context is good to put in commit messages and PR descriptions. As a code reviewer my assumption is that the proposed code is complete.

Maybe we leave some TODOs in code to remind us that this is temporary?